business/istio_validations.go

```diff
@@ -63,23 +63,20 @@ func (in *IstioValidationsService) GetValidations(ctx context.Context, namespace
 	errChan := make(chan error, 1)
 
 	var istioConfigList models.IstioConfigList
-	var exportedResources kubernetes.ExportedResources
 	var services models.ServiceList
 	var namespaces models.Namespaces
 	var workloadsPerNamespace map[string]models.WorkloadList
 	var mtlsDetails kubernetes.MTLSDetails
 	var rbacDetails kubernetes.RBACDetails
 	var registryServices []*kubernetes.RegistryService
 
-	wg.Add(8) // We need to add these here to make sure we don't execute wg.Wait() before scheduler has started goroutines
+	wg.Add(6) // We need to add these here to make sure we don't execute wg.Wait() before scheduler has started goroutines
 
 	// We fetch without target service as some validations will require full-namespace details
-	go in.fetchIstioConfigList(ctx, &istioConfigList, namespace, errChan, &wg)
-	go in.fetchExportedResources(ctx, &exportedResources, namespace, errChan, &wg)
+	go in.fetchIstioConfigList(ctx, &istioConfigList, &mtlsDetails, &rbacDetails, namespace, errChan, &wg)
 	go in.fetchNamespaces(ctx, &namespaces, errChan, &wg)
 	go in.fetchAllWorkloads(ctx, &workloadsPerNamespace, errChan, &wg)
```

Comment:
Another comment that I think it's worth reviewing in this work.
At this level with have a "fetchNamespaces" and a "fetchAllWorkloads", but inside the "fetchAllWorkloads" we are repeating the "GetNamespaces".

I think that's something we can review if we are not duplicating calls.

Reply:
gree, fetchAllWorkloads could also return namespaces